int32[6] mac_adress
int32 drone_mode
int32 drone_filter
bool is_armed
float32 voltage 
float32 current 
